### PR TITLE
Event hub igot UI

### PR DESCRIPTION
**Pull request by keyurmeandev**

### CODE REVIEW

File: apps/fusion/project/ws/app/src/lib/routes/events/routes/event-detail/event-detail.component.html [hunk 1]
```diff
@@ -85,10 +85,23 @@
   </ng-container>
 
   <ng-container *ngIf="currentFilter == 'participants'">
-      <div class="flex" style="width: 65%;">
+
+      <div class="flex" style="width: 65%;" *ngIf="participantsCount > 0">
```
Comment (christyfernandes) [inline apps/fusion/project/ws/app/src/lib/routes/events/routes/event-detail/event-detail.component.html]: Remove inline styling

File: apps/fusion/project/ws/app/src/lib/routes/events/routes/event-detail/event-detail.component.ts [hunk 2]
```diff
@@ -73,13 +74,16 @@ export class EventDetailComponent implements OnInit {
...
             this.eventDataObj = responseObj.result[0]
             this.overviewData.push(responseObj.result[0])
-            Object.keys(responseObj.result[0].creatorDetails).forEach((index: any) => {
-                const obj = {
-                    name: responseObj.result[0].creatorDetails[index].name,
-                    id: responseObj.result[0].creatorDetails[index].id,
-                }
-                this.participantsData.push(obj)
-            })
+            if (responseObj.result[0].creatorDetails !== undefined && responseObj.result[0].creatorDetails.length > 0) {
```
Comment (christyfernandes) [inline apps/fusion/project/ws/app/src/lib/routes/events/routes/event-detail/event-detail.component.ts]: This will fail if the result is empty array

**Review by christyfernandes — CHANGES_REQUESTED** [review]

**Comment by keyurmeandev** [comment]
Okay. I will review it and then do the commit.

File: apps/fusion/project/ws/app/src/lib/routes/events/routes/event-detail/event-detail.component.html (hunk 1 above)
Comment (keyurmeandev) [reply apps/fusion/project/ws/app/src/lib/routes/events/routes/event-detail/event-detail.component.html]: This is done

File: apps/fusion/project/ws/app/src/lib/routes/events/routes/event-detail/event-detail.component.ts (hunk 2 above)
Comment (keyurmeandev) [reply apps/fusion/project/ws/app/src/lib/routes/events/routes/event-detail/event-detail.component.ts]: I have made the change for the same.

**Pull request merged**